### DISCUSSION TITLE
Images in 10.0.5 are not showing in safari(on every other device!)

### DISCUSSION THREAD

**Discussion by royjosefsson** [vercel/next.js#22014] [Help] [answered]
**What version of Next.js are you using?**

10.0.5

**What version of Node.js are you using?**

14.15.4

**What browser are you using?**

Safari

**What operating system are you using?**

Linux Ubuntu 20.04

**How are you deploying your application?**

npm run build && pm2 start server.js --node-args=\"--max-old-space-size=4096\""

**Describe the Bug**

Optimized images are not showing. It's different from device to device.

(I'm not sure the issue is because the images are inside a 'next/link', no?)

Some devices are showing all images
Some devices are only showing pngs, jpeg, jpg and not svgs.
Some devices are not showing any images what so ever.

**Expected Behavior**

Images to be displayed

**To Reproduce**

Create an image using 'next/image

**Here are some logs from an iMac:**
![20210209_173512](https://user-images.githubusercontent.com/23550449/107438643-17b15800-6b31-11eb-98c3-1b72fd390dbf.jpg)

**Images on an iPhone:**(some are showing?)
![Image from iOS](https://user-images.githubusercontent.com/23550449/107438769-5a733000-6b31-11eb-99c1-a7f069c7b2b7.jpg)

**A mac, working just fine:**
<img width="1440" alt="Screenshot 2021-02-09 at 9 29 38 PM" src="https://user-images.githubusercontent.com/23550449/107438832-770f6800-6b31-11eb-8fb0-79558e94992c.png">

The site works flawlessly on pc and android.

**Comment by royjosefsson** [comment, marked as answer]
Site: [https://jiroy.com](https://jiroy.com)

I've found the issue.
The issue is that: 
Safari 3.1 - 13.1 and iOS Safari 3.2 - 13.7 isn't compatible with [webp](https://developers.google.com/speed/webp).
[https://caniuse.com/webp](https://caniuse.com/webp)

"next/image" optimizes images to webp on runtime(if the browser supports it*).
The reason why an uncompatible iOS device even is trying to show a webp-image remains unclear.

I'm going to try and disable webp for now, or optimize the images in a different way

Edit: This is a duplicate of [this link](https://github.com/vercel/next.js/issues/20794)

**Reply by jaddoescad** [reply]
Have you been able to find a quick fix?

**Comment by styfle** [comment]
This has been fixed in https://github.com/vercel/next.js/pull/35190

Please give it a try with `npm i next@canary` or `yarn add next@canary`, thanks!
